base/tuple.jl

```diff
@@ -52,7 +52,7 @@ julia> Base.setindex((1, 2, 6), 2, 3) == (1, 2, 2)
 true
 ```
 """
-function setindex(x::Tuple, v, i::Integer)
+public function setindex(x::Tuple, v, i::Integer)
```

Comment:
This syntax is incorrect; `public` is similar to `export` in that its syntax is `public <one or more names delimited by commas>`. Public, unexported Base functions go in the list in `base/public.jl`.